omniscidb/Shared/Config.h

```diff
@@ -83,6 +83,7 @@ struct CodegenConfig {
   bool null_mod_by_zero = false;
   bool hoist_literals = true;
   bool enable_filter_function = true;
+  short dump_after_all_severity{0};
```

Comment:
I'd propose to move it to debug config structure. Also, rename it to `dump_llvm_ir_after_each_pass` or similar, simple `dump` is too generic since we have different IRs on various levels.